src/components/Accounts/Menu.css

```diff
@@ -0,0 +1,9 @@
+@import "@folio/stripes-components/lib/variables";
+
+.firstMenuLastItem {
```

Comment:
Would it be possible to use a `:last-of-type` of `:last-child` selector with the `.firstMenuItems` selector instead of this separate class? It'd minimise the risk of accidentally marking the wrong element as `firstMenuLastItem`